WORKSPACE.bazel

```diff
@@ -0,0 +1,76 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "aspect_bazel_lib",
+    sha256 = "229a6d65b8b30af0dcaeb77c723b568ef7a2e4ad54dd65168a16992b6b6fe4c7",
+    strip_prefix = "bazel-lib-1.12.0",
+    url = "https://github.com/aspect-build/bazel-lib/archive/refs/tags/v1.12.0.tar.gz",
+)
+
+load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies")
+
+aspect_bazel_lib_dependencies()
+
+http_archive(
+    name = "aspect_rules_js",
+    sha256 = "515277ae357e62f52e29e0bfb60b73d2d062b8d00d21351d31f37c5bb275d4f5",
+    strip_prefix = "rules_js-1.5.3",
```

Comment:
Some nice per improvements have landed in rules_js since this release. Best to update to the latest of rules_js and  aspect_bazel_lib before landing.